### ISSUE TITLE
ARIA attribute cannot be used, add a role attribute or use a different element: aria-label

### ISSUE THREAD

**Issue by danchristensen2000** [jackocnr/intl-tel-input#1449]
We upgraded to the latest version of the component, and we are now getting accessibility errors.  We use axe tooling to check accessibility.  I believe the error is indicating that 'aria-label' is not valid.  The problem appears to have been introduced in this commit from Apr 6.  https://github.com/jackocnr/intl-tel-input/commit/71dfdd4bd84eb2d6ba857f54d6e6e3bae072a58c

### Steps to reproduce
Upgrade to any version >= 18.0.0.  Test accessibility on a page using axe.

### Expected behaviour
There should not be any accessibility errors

### Actual behaviour
Axe reports the following

Expected DIV 'root279' to be accessible, but there were 1 errors:
SERIOUS [wcag2a] aria-allowed-attr VIOLATION: Elements must only use allowed ARIA attributes: Ensures ARIA attributes are allowed for an element's role
[More information: https://dequeuniversity.com/rules/axe/4.2/aria-allowed-attr?application=axeAPI](https://dev.azure.com/nbsdev/Services/_git/822af853-c4aa-433b-8df7-37c1be213b74?path=%2Fdequeuniversity.com%2Frules%2Faxe%2F4.2%2Faria-allowed-attr%3Fapplication%3DaxeAPI&version=GBdan%2FupdateTelephoneInputPackages&_a=contents&line=1&lineEnd=2&lineStartColumn=1&lineEndColumn=1&lineStyle=plain)
Targets ["div[aria-controls=\"iti-69__country-listbox\"]"]
Fix all of the following:
ARIA attribute cannot be used, add a role attribute or use a different element: aria-label
Targets ["div[aria-controls=\"iti-70__country-listbox\"]"]
Fix all of the following:
ARIA attribute cannot be used, add a role attribute or use a different element: aria-label
Targets ["div[aria-controls=\"iti-71__country-listbox\"]"]
Fix all of the following:
ARIA attribute cannot be used, add a role attribute or use a different element: aria-label

**Comment by jackocnr** [comment]
Hmm I'm confused by this. aria-label is [a valid ARIA attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label). In the plugin we give 2 elements an aria-label and both also have a role attribute, so I don't understand that error message either. Do you have any ideas of what we should do to address this? Or perhaps it is a bug in Axe?

**Comment by danchristensen2000** [comment]
I looked closer at the source, and the attribute for role is not on the div on my page, but it is there on your demo page.  So, there must be something we are doing that is causing a problem.  Sorry for the confusion.  

![image](https://github.com/jackocnr/intl-tel-input/assets/55195676/a513c2a7-d796-4dfd-a172-93258233816e)
